gitkraken 3.1.2

parkiyong opened the pull request:
This release has dependency to [libcurl-gnutls](https://github.com/wrajaka/kcp-libcurl-gnutls) that need to be manually build. This information is added in README file.

parkiyong committed:
gitkraken 3.1.2

fjmorazan commented:
It's not neccesary to add that depency on the PB nor advice it to install on the README, for that reason it's available curl-kcp, it's already added a symlink to /urs/lib to make it work https://github.com/KaOS-Community-Packages/curl-kcp/commit/7f831f8372feca4146d038f356230f77dcccafc2#diff-dfa7cdd9c34798e1af82f172b0822f08R39
So you should remove that dependency and change the file [gitkraken.sh](https://github.com/KaOS-Community-Packages/gitkraken/blob/master/gitkraken.sh) to point to the new symlink, to make it work again 😅

fjmorazan commented:
Hi @wrajaka,
It's not necessary to add that depency on the PB nor advice to install it on the README, for that reason it's available **curl-kcp** on KCP, and it was [added](https://github.com/KaOS-Community-Packages/curl-kcp/commit/151070329cd85e6b00426f76f001ee9fc27c1420#diff-dfa7cdd9c34798e1af82f172b0822f08R39) today a **symlink** to /urs/lib to make it work
So you should remove that dependency and change the file [gitkraken.sh](https://github.com/KaOS-Community-Packages/gitkraken/blob/master/gitkraken.sh) _to point to the new symlink_ (**/usr/lib/libcurl-gnutls.so.4** instead of  **/opt/curl/lib/libcurl.so**), to make it work again 😅

parkiyong committed:
gitkraken 3.1.2

Remove dependency to libcurl-gnutls. Clear README. Change symlink in gitkraken.sh to point to correct libcurl-gnutls

parkiyong commented:
Ok. I will create another pull request for above suggestion.
